src/IndexRange/IndexRange.csproj

```diff
@@ -1,8 +1,8 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I wonder if `.gitattributes` could be pursuaded to strip BOM...

Reply:
I don't know where that came from. Deleted now.